### DISCUSSION TITLE
SyntaxError: invalid syntax `:=`

### DISCUSSION THREAD

**Discussion by india666661** [Steffo99/greed#189] [Q&A] [answered]
Generate the configuration file，
when executed **$ python -OO core.py**，The following problems occur：

 File "core.py", line 79
    if db_engine := os.environ.get("DB_ENGINE"):
                  ^
SyntaxError: invalid syntax

Can anyone help me, thank you very much！

**Comment by Steffo99** [comment, marked as answer]
You're using an outdated Python version.

Please update to [the latest version of Python](https://www.python.org/downloads/) to fix the problem!
